addon/helpers/find-by.js

```diff
@@ -29,7 +29,7 @@ export default Helper.extend({
       let array = get(this, 'array');
       let value = get(this, 'value');
 
-      return array.findBy(byPath, value);
+      return array.find(e => get(e, byPath) === value);
```

Comment:
I think it's preferable to wrap `array` in `emberArray` and then just use `findBy`.